src/librustc_trans/adt.rs

```diff
@@ -135,11 +131,9 @@ pub fn finish_type_of<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>,
 fn generic_type_of<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>,
                              t: Ty<'tcx>,
                              name: Option<&str>,
-                             sizing: bool,
-                             dst: bool) -> Type {
```

Comment:
I removed the dst parameter because it was always set to false, seemed like it was no longer used.